### PR TITLE
[IMPROVEMENT] Add support for setLaunchURLsInApp

### PR DESCRIPTION
**Pull request by shepherd-l**
### Changed
- Added support for `setLaunchURLsInApp`

---
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459)

### CODE REVIEW

**Review by jkasten2 — APPROVED** [review]
One comment on the change log, otherwise this looks good to me. After fixing it feel free to merge without another approval from my end.

> Reviewed 7 of 7 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459#-:-MxuToGt11RGQogd_jv3:b7vq76q)* status: all files reviewed, 1 unresolved discussion (waiting on @shepherd-l)

___
*[OneSignalExample/Assets/OneSignal/CHANGELOG.md, line 10 at r1](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459#-MxuTr4xC5Hjw6xawcO4:-MxuTr4xC5Hjw6xawcO5:bis6otr) ([raw file](https://github.com/onesignal/onesignal-unity-sdk/blob/0ba1a9333cd8c140fc1b2efe8e31dc1070172465/OneSignalExample/Assets/OneSignal/CHANGELOG.md#L10)):*
> ```Markdown
> ### Changed
> - Updated VSP Attribution with the latest version of script
> - Added support for `setLaunchURLsInApp`
> ```

I would include this is for iOS only somewhere in this line.

**Review by jkasten2 — CHANGES_REQUESTED** [review]
Looks like we added a default app_id by mistake we should remove.

> Reviewed 9 of 9 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459#-:-Myhk9wQ5pLw7frVc99Y:b-2p0sul)* status: all files reviewed, 3 unresolved discussions (waiting on @shepherd-l)

___
*[OneSignalExample/Assets/OneSignal/Example/OneSignalExampleBehaviour.cs, line 430 at r2](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459#-Myhk0u1CJQyz4FPiJPX:-Myhk0u1CJQyz4FPiJPY:b-anl82s) ([raw file](https://github.com/onesignal/onesignal-unity-sdk/blob/679e53c0be022e502de3c7a28d58e44c4283bb71/OneSignalExample/Assets/OneSignal/Example/OneSignalExampleBehaviour.cs#L430)):*
> ```Smalltalk
>      */
> 
>     public void ToggleLaunchURLsInApp() {        
> ```

nit, extra spaces to the right of the open bracket `{`.
___
*[OneSignalExample/Assets/OneSignal/Example/OneSignalExampleScene.unity, line 1436 at r2](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459#-MyhjFbB6GGjf1e2Figh:-MyhjFbB6GGjf1e2Figi:b-ulyoog) ([raw file](https://github.com/onesignal/onesignal-unity-sdk/blob/679e53c0be022e502de3c7a28d58e44c4283bb71/OneSignalExample/Assets/OneSignal/Example/OneSignalExampleScene.unity#L1436)):*
> ```
>   m_LocalPosition: {x: 0, y: 0, z: 0}
>   m_LocalScale: {x: 0.9, y: 0.9, z: 1}
>   m_ConstrainProportionsScale: 0
> ```

This looks it was added by Unity automatically as part of an upgrade. This is fine but in the future it should be its own commit (or in some cases it's own PR if it is larger) so it is easier to see the intended changes.
___
*[OneSignalExample/Assets/OneSignal/Example/OneSignalExampleScene.unity, line 1535 at r2](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459#-Myhj1E13p1meiK5HHQw:-Myhj1E13p1meiK5HHQx:b-dwut5f) ([raw file](https://github.com/onesignal/onesignal-unity-sdk/blob/679e53c0be022e502de3c7a28d58e44c4283bb71/OneSignalExample/Assets/OneSignal/Example/OneSignalExampleScene.unity#L1535)):*
> ```
>   externalId: 
>   phoneNumber: 
>   appId: ba9d1c88-4130-498c-849f-9a4bf36ec0b8
> ```

Remove app_id value.

**Review by jkasten2 — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459#-:-MyhqeJBBKzgkxsqKl8Y:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @shepherd-l)

File: com.onesignal.unity.core/Runtime/OneSignal.cs
Comment (adamschlesinger) [inline com.onesignal.unity.core/Runtime/OneSignal.cs:178]: Can we get this marked as iOS only with a remarks tag? ie 
```/// <remarks>iOS Only</remarks>```

**Review by adamschlesinger — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459)* status: all files reviewed, 2 unresolved discussions (waiting on @shepherd-l)

___
*[com.onesignal.unity.core/Runtime/OneSignal.cs, line 179 at r3](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459#-N--_1FbDMOmvLOZKgJ-:-N--_1FbDMOmvLOZKgJ0:b-jzzb9c) ([raw file](https://github.com/onesignal/onesignal-unity-sdk/blob/66cf6012615c67c11465ea74d75002aff6a47345/com.onesignal.unity.core/Runtime/OneSignal.cs#L179)):*
> ```Smalltalk
>         /// <see cref="Initialize"/> is called.
>         /// </summary>
>         public abstract void SetLaunchURLsInApp(bool launchInApp);
> ```

Can we get this marked as iOS only with a remarks tag? ie
```/// <remarks>iOS Only</remarks>```

**Review by adamschlesinger — APPROVED** [review]
> Reviewed 6 of 9 files at r2, 2 of 2 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459)* status: all files reviewed, 1 unresolved discussion (waiting on @shepherd-l)

**Review by adamschlesinger — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/onesignal/onesignal-unity-sdk/459#-:-N-4Qvdy4xtBW2RhS-Ht:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @shepherd-l)

**Pull request merged**
